Unit3/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <tab id="rltsquare" translate="label" sortOrder="10">
+            <label>RLTSquare</label>
+        </tab>
+        <section id="unit3" translate="label" sortOrder="130" showInDefault="1" showInWebsite="1" showInStore="1">
+            <class>separator-top</class>
+            <label>Unit 3</label>
+            <tab>rltsquare</tab>
+            <resource>RLTSquare_Unit3::unit3_config</resource>
+            <group id="general" translate="label" type="text" sortOrder="10" showInDefault="1" showInWebsite="0"
+                   showInStore="1">
+                <label>Settings</label>
+                <field id="enable" translate="label" type="select" sortOrder="1" showInDefault="1" showInWebsite="0"
+                       showInStore="1">
+                    <label>Module Enable</label>
+                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                </field>
+                <field id="username" translate="label" type="text" sortOrder="1" showInDefault="1" showInWebsite="0"
+                       showInStore="1">
+                    <label>UserName</label>
+                    <comment>Enter Username</comment>
+                </field>
+                <field id='password' translate='label' type='password' sortOrder='3' showInDefault='1' showInWebsite='1'
```

Comment:
Password and username fields should have been set for each environment ... like staging_password, production_password ... then use the <depends> to hide/show it based of environment type. We want it so that when we switch environment, we have the user/pass stored specific to each env. The helper functions such as `getPassword` can then just return pass based of chosen env.